appeals/e2e/cypress/page_objects/caseDetailsPage.js

```diff
@@ -201,6 +209,8 @@ export class CaseDetailsPage extends Page {
 			month: 'long',
 			year: 'numeric'
 		}).format(dateToday); // Format the date
+		this.elements
+			.getAppealStartDate()
 		cy.get('.appeal-start-date > .govuk-summary-list__value')
```

Comment:
I'd suggest pulling out this inline selector and adding it to the elements collection on line 35 of this file